traversals/morris_traversal.py

```diff
@@ -0,0 +1,51 @@
+class Node:
+    def __init__(self, data):
+        self.data = data
+        self.left = None
+        self.right = None
+
+
+def Morris_traversal(root):
+    """
+    Morris(InOrder) travaersal is a tree traversal algorithm that does not employ
+     the use of recursion or a stack. In this traversal, links are created as
+    successors and nodes are printed using these links.
+    Finally, the changes are reverted back to restore the original tree.
+    """
+    InorderTraversal = []
+    # set current to root of binary tree
+    curr = root
```

Comment:
```suggestion
    current = root
```
Explicit is better than implicit
Please make this change wherever the variable was used.